inst/htmlwidgets/datatables.js

```diff
@@ -50,7 +50,16 @@ var transposeArray2D = function(a) {
 HTMLWidgets.widget({
   name: "datatables",
   type: "output",
-  renderValue: function(el, data) {
+  initialize: function(el, width, height) {
+    $(el).text('TO BE RENDERED...');
```

Comment:
I can also go without this line, in which case it seems I only need to check `if (el.offsetWidth === 0)` below on line 58, because `height` is still 0 when the resize method is triggered since el is empty.